### PR TITLE
SPRT-94 (elimination-generic-text-and-add-smooth-scroll-a-button)

### PR DESCRIPTION
**Pull request by GonzaEnrique27**
The generic text that was below the main view has been removed and a smooth-scroll button has been implemented to redirect to the form at the end of the app.

### CODE REVIEW

**Review by eddymdz — CHANGES_REQUESTED** [review]
@GonzaEnrique27 you have 11 changes only for text changes. Please check

File: layouts/partials/content/hero.html [hunk 1]
```diff
@@ -6,15 +8,18 @@
...
-  <a href="#" class="button" target="_blank">APRENDE MÁS</a>
+  <div class="relative md:w-[800px] mx-auto"> <p class="text-center font-extrabold text-2xl md:text-3xl my-2"> NOSOTROS</p>
+    <p class="text-lg md:text-xl px-8 md:px-0 text-justify">Somos una consultora de innovación tecnológica y transformación digital que ayuda a organizaciones y particulares en la producción de software de calidad de forma eficiente, sostenible y divertida.
 
-  <div class="relative md:w-[800px] mx-auto">
-    <p class="text-3xl md:text-4xl text-center font-extrabold my-4">LOREM IPSUM</p>
-    <p class="text-lg md:text-xl px-8 md:px-0 text-justify">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quis tenetur reiciendis officia laborum distinctio ea amet earum? Vel, minus maxime sed soluta qui non adipisci corrupti natus nobis itaque pariatur.</p>
+      Queremos formar parte de la construcción tecnológica de las ideas del futuro que también contribuyan al desarrollo humano y material de las personas involucradas en el proceso.</p>
   </div>
+  <a href="#smooth-scroll" class="button" style="margin-top: 20px;">APRENDE MÁS</a>
```
Comment (njrzr) [inline layouts/partials/content/hero.html:22]: Why are using an inline style, you should use tailwind to style the content of the site.

File: static/styles/style.css [hunk 2]
```diff
@@ -50,4 +51,70 @@
...
   background: radial-gradient(#f3d02b, #e38718);
 }
-/*#endregion*/
\ No newline at end of file
+/*#endregion*/
+
+
+
+/* Validaciones */
+
```
Comment (njrzr) [inline static/styles/style.css:59]: @GonzaEnrique27 you should use tailwind to style the content of the site, explain me why did you need to use css.
